vanilla.js

```diff
@@ -0,0 +1,38 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    let page = document.getElementById("page");
```

Comment:
Hey Lorenzo, am I supposed to reply to these here? If so:

1. I do not expect it to be reassigned, I generally use let instead of const but would have no problem using const for variables that aren't going to be changed if that is best practice.

2. I intended to pass the page element into the "addPost" function, but after doing a quick search it seems that JS automatically creates a variable for HTML id's named according to that id. So the "page" being used in "addPost" was referencing that auto-created variable and functioning properly, which is why I didn't notice that I wasn't using the "page" variable I created and intended to use.